discovery/db_discovery.go

```diff
@@ -212,3 +221,21 @@ func pmTicketParams(params *net.TicketParams) *pm.TicketParams {
 		Seed:              new(big.Int).SetBytes(params.Seed),
 	}
 }
+
+func (dbo *DBOrchestratorPoolCache) PollOrchestratorInfo(bcast common.Broadcaster) {
+	ticker := getTicker()
+	go func() {
+		for range ticker.C {
```

Comment:
Since the first tick will not occur until after `cacheRefreshInterval`, we will not cache any off-chain info until after `cacheRefreshInterval`. The [old code](https://github.com/livepeer/go-livepeer/blob/master/discovery/db_discovery.go#L37) cached off-chain info *before* starting the ticker loop - we could do the same here.

Reply:
fixed in 9458ef19